test/pom.xml

```diff
@@ -198,6 +198,14 @@
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-params</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-suite-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-suite-engine</artifactId>
+    </dependency>
```

Comment:
Are these compile time dependencies or runtime dependencies? It seems like maybe the engine is a runtime dependency. It probably should have a runtime scope.